src/chem_lex/ChemlabTokens.py

```diff
@@ -4,9 +4,50 @@
     'then': 'then',
     'TRUE': 'TRUE',
     'FALSE': 'FALSE',
-    'detail': 'detail'
+    'detail': 'detail',
+    'convert': 'convert',
+    'form': 'form',
+    'balanced?': 'isBalanced',
 }
 
+unit_prefix = {
+    'G': 'PrefixTok', # GIGA
+    'M': 'PrefixTok', # MEGA
+    'k': 'PrefixTok', # KILO
+    'h': 'PrefixTok', # HECTOR
+    'da': 'PrefixTok', # DEKA
+    'd': 'PrefixTok', # DECI
+    'c': 'PrefixTok', # CENTI
+    'm': 'PrefixTok', # MILLI
+    'u': 'PrefixTok', # MICRO
+    'n': 'PrefixTok', # NANO
+    'p': 'PrefixTok', # PICO
+    'f': 'PrefixTok', # FEMTO
+}
+
+supported_units = {
+    'ft': 'UnitTok', # FEET
+    'me': 'UnitTok', # METERS
```

Comment:
The abbreviation for meter should be "m", instead of "me".

Reply:
Problem is we already have "m" alone as milli in the prefixes so they must be different. That's why I used "me" as meters. We could change to something else but it must be unique so we cant use "m" unless we change the name of milli to another thing